docs/unified-test-documentation/dasharo-compatibility/320-update-firmware.md

```diff
@@ -0,0 +1,108 @@
+# Dasharo Compatibility: Update Firmware
+
+## Test cases
+
+### Common
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+1. Install the `dmidecode` package: `sudo apt install dmidecode`.
```

Comment:
Also we should add the following point:

Make yourself familiar with https://docs.dasharo.com/variants/clevo_nv41/fwupd_usage/

Reply:
added